Examples/test-suite/arrays.i

```diff
@@ -67,10 +68,52 @@ void array_pointer_func(int (*x)[10]) {}
 %inline %{
 typedef float FLOAT;
 
-typedef FLOAT cartPosition_t[3]; 
+typedef FLOAT cartPosition_t[3];
 
 typedef struct {
 cartPosition_t p;
 } CartPoseData_t;
 
 %}
+
+#ifdef SWIGFORTRAN
+/* Fortran can't handle hex, but it can substitute compile-time constants like
+ * the enum into the 'contains' part of the code. HOWEVER, the interface would
+ * have to "import" the enums for that to work, which we don't support. */
```

Comment:
Yes, because Fortran has the ability to natively understand dimensioned arrays, I have SWIG treat them distinctly from undimensioned pointers. Should I add a note in that section of the manual mentioning the exception in the Fortran target language?